pkgs/by-name/su/sunshine/package.nix

```diff
@@ -94,6 +94,8 @@ stdenv'.mkDerivation rec {
     ]
     ++ lib.optionals cudaSupport [
       autoAddDriverRunpath
+      cudaPackages.cuda_nvcc
+      (lib.getDev cudaPackages.cuda_cudart)
```

Comment:
Note: `mkDerivation` automatically selects `getDev`.
Question: do you know how exactly the build platform's `cudart` is used and why host platform's `cudart` is insufficient?